operator/pkg/central/values/translation/translation.go

```diff
@@ -91,7 +91,7 @@ func (t Translator) translate(ctx context.Context, c platform.Central) (chartuti
 	}
 
 	monitoring := c.Spec.Monitoring
-	v.AddChild("monitoring", globalMonitoring(monitoring))
+	v.AddChild("monitoring", getGlobalMonitoring(monitoring))
```

Comment:
the title of the pr is for "securedCluster". Is that an error ?

Reply:
I've changed that to be consistent with secured Cluster per your request https://github.com/stackrox/stackrox/pull/7036#discussion_r1277391854